docs/docs/migration.md

```diff
@@ -1,35 +1,35 @@
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; For more information about the `supported-platforms` build parameter, see [Configuring and Packaging Modules According to Target Platforms](configuration.md#configuring-and-packaging-modules-according-to-target-platforms)
   
-* The following `build-rarameters` are not supported by the Cloud MTA Build Tool: <ul><li>`npm-opts`<li>`grunt-opt`<li>`maven-opts`</ul>
+* The following `build-parameters` are not supported by the Cloud MTA Build Tool: <ul><li>`npm-opts`<li>`grunt-opt`<li>`maven-opts`</ul>
 
-  If you need to change default build behaviour defined for the correponding builder, please use [configure `custom` builder](configuration.md#configuring-the-custom-builder)
-  For complete list of available builders and their default behaviour, see [Builders execution commands](https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/builder_type_cfg.yaml).
+  If you need to change the default build behavior defined for the correponding builder, see [configure `custom` builder](configuration.md#configuring-the-custom-builder).
+  For a complete list of available builders and their default behaviors, see [Builders execution commands](https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/builder_type_cfg.yaml).
   <br>
 
 #### New features in the Cloud MTA Build Tool
 
-* If you would like to run some builder process before running builders of the specific modules, define it in [global `before-all` build parameters](configuration.md#configuring_before_all_build)
-* You can define your own build commands by [configuring `custom` builder](configuration.md#configuring-the-custom-builder)
+* If you want to run a builder process before running builders of the specific modules, define it according to [global `before-all` build parameters](configuration.md#configuring_before_all_build).
+* You can define your own build commands as described here: [configuring `custom` builder](configuration.md#configuring-the-custom-builder).
 
 
-#### Features that are temporary not available in the Cloud MTA Build Tool
+#### Features that are temporarily not available in the Cloud MTA Build Tool
 
-The following features are supported by the [Multitarget Application Archive Builder](https://help.sap.com/viewer/58746c584026430a890170ac4d87d03b/Cloud/en-US/ba7dd5a47b7a4858a652d15f9673c28d.html) and will be provided in the Cloud MTA Build Tool soon.
+The following features are supported by the [Multitarget Application Archive Builder](https://help.sap.com/viewer/58746c584026430a890170ac4d87d03b/Cloud/en-US/ba7dd5a47b7a4858a652d15f9673c28d.html) and will be provided in the Cloud MTA Build Tool soon:
 
-* MTA build with extension files
-* Configuring Timeout Sessions
-* Configuring the Build Artifact Name
+* Generation of MTA builds with extension files
```

Comment:
"Configuration" is not applicable for "MTA builds". I suggest "Running MTA builds..." I will change it to the original wording until you propose an alternative.

Reply:
Actually I wrote "Generation of MTA builds" -  is that what you are referring to?  
How about "Running MTA builds with extension files"?
Are the other uses of "configuration" here ok?

Reply:
"Running MTA builds with extension files" is perfect. Other uses of "configuration" are absolutely correct. I will update.

Reply:
Great, thanks.